l10n_be_cooperator/migrations/16.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,40 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.column_exists(
+        env.cr, "tax_shelter_declaration", "tax_shelter_type"
+    ):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            ALTER TABLE tax_shelter_declaration
+            ADD COLUMN tax_shelter_type character varying
+            """,
+        )
+
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE tax_shelter_declaration
+            SET tax_shelter_type = 'scale_up'
+            WHERE tax_shelter_declaration.tax_shelter_percentage = '25'
```

Comment:
one update statement is enough:

```suggestion
            UPDATE tax_shelter_declaration
            SET tax_shelter_type = CASE
                WHEN tax_shelter_percentage = '25' THEN 'scale_up'
                WHEN tax_shelter_percentage = '30' THEN 'start_up_small'
                WHEN tax_shelter_percentage = '45' THEN 'start_up_micro'
            END
```